File: Lua_5.4/2749 - Saída 3.lua
--- author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
--- copyright: Josélio Júnior (Lunatic Fox) 2022

local b, s, w = ('-'):rep(39), ('|%s|\n'):format((' '):rep(37)), 'x = 35'
local l = {
  ('|%s%s|\n'):format(w, (' '):rep(31)),
  ('|%s%s%s|\n'):format((' '):rep(15), w, (' '):rep(16)),
  ('|%s%s|\n'):format((' '):rep(31), w)
}
print(('%s\n%s%s%s%s%s%s'):format(b, l[1], s, l[2], s, l[3], b))

File: C#_mono_5.10.1.20/2749 - Saída 3.cs
/// author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
/// copyright Josélio Júnior (Lunatic Fox) 2022

using System;
using System.Linq;

class URI {
  static void Main(string[] args) { 
    string b = new string(' ', 39).Replace(" ", "-");
    string i = $"|{new string(' ', 37)}|\n";
    string[] w = new string[] {
      $"x{ new string(' ', 31) }",
      $"{ new string(' ', 15) }x{ new string(' ', 16) }",
      $"{ new string(' ', 31) }x"
    }.Select(e => $"|{ e.Replace("x", "x = 35") }|\n")
    .ToArray();

    Console.WriteLine($"{b}\n{string.Join(i, w)}{b}");
  }
}
